micro-rdk/src/common/grpc.rs

```diff
@@ -154,28 +148,20 @@ where
 }
 
 impl<'a> GrpcServerInner<'a> {
-    fn encode_message_into<M: Message>(m: M, mut buffer: BytesMut) -> Result<Bytes, ServerError> {
+    fn get_encoded_message<M: Message>(m: M) -> Result<Bytes, ServerError> {
```

Comment:
I think I like the old name better. This method isn't getting an encoded message from somewhere, it is taking a message and encoding it.

Since you no longer need the "new buffer" vs "internal buffer" distinction with the three existing methods were intended to support and now only need the former, you could just call this method `encode_message`.